core/contextmenu.ts

```diff
@@ -58,6 +58,15 @@ export function setCurrentBlock(block: Block | null) {
  */
 let menu_: Menu | null = null;
 
+/**
+ * Gets Menu instance.
+ *
+ * @returns menu instance.
```

Comment:
One thing that would be really helpful to include is context on the menu's lifecycle. Specifically:
- When is it defined? (Relative to other functions called here)
- When is `null` returned? What does that mean/what can the caller infer from receiving a null value?
- Is it valid to save the returned value or could that cause problems (e.g. if the menu is later closed)?

These would be great to clarify in the documentation here for a more solid defined API contract for callers.